docs/content/src/learn/guides/services/converting-json-to-xml-and-upload-to-ftp/README.md

```diff
@@ -0,0 +1,157 @@
+# Convert JSON to XML and upload to FTP
+
+## About 
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the support of connectors. In this guide, we are mainly focusing on connecting to a file server with the FTP connector. You can find other integrations modules from the [wso2-ballerina](https://github.com/wso2-ballerina) Github repository.
+
+The FTP connector support as listener where listen to given remote location and trigger when file added or deleted. Also, it supports to connect to a file server as the client and perform various operations such as get, put, delete, etc. The FTP connector can be used to connect to your remote file server with FTP, SFTP, FTPS protocols.
+
+## What you'll build
+The Ballerina has first-class support for HTTP and implementing an HTTP service is straightforward. The caller will send JSON payload that consists of employees data. The HTTP service read the JSON payload and transform to XML. Then with the support of FTP connector, the XML employees data upload to the FTP server.
+
+## Prerequisites
+Link to download ballerina integrator.
+
+## Implementation
+The ballerina project should create for the integration use case explained above. Please follow the steps given below. You can learn about the ballerina project and module in this link.
+
+1. Create a project
+```bash
+$ ballerina new converting-json-to-xml-and-upload-to-ftp
+```
+Change directory to the converting-json-to-xml-and-upload-to-ftp.
+
+2. Add a module
+```bash
+$ ballerina add uploadtoftp
+```
+
+The project structure should look like below.
+```shell
+converting-json-to-xml-and-upload-to-ftp
+├── Ballerina.toml
+└── src
+    └── uploadtoftp
+        ├── main.bal
+        ├── Module.md
+        ├── resources
+        └── tests
+            └── resources
+```
+
+3. Write the integration
+You can open the project with VSCode. The integration implemention going to write in the `main.bal` file. 
+
+<!-- INCLUDE_CODE: src/uploadtoftp/main.bal -->
+
+The `ftpConfig` created providing file server credentials. Here it reads from the config file. You have to create a file called `ballerina.conf` under the root path of the project structure. The file should have the following configurations.
+
+```
+[ftp]
+  host="<file_server_url>"
+  port=<file_server_port>
+  username="<file_server_username>"
+  password="<file_server_password>"
+```
+
+The username and password are sensitive data. Usually, we don't expose those data to the outside. The ballerina supports encrypting sensitive data and uses them in the program.
+
+```shell
+$ ballerina encrypt
+```
+
+The execution of the encrypt action will ask for value and secret key. Once you provide input, it'll output the encrypted value that can directly use in the config file.
```

Comment:
```suggestion
When you execute the encrypt action, it will ask for the value and secret key. Once you input this, it will provide the encrypted value that can directly use in the config file.
```